server/src/main/java/org/spine3/server/entity/FieldMasks.java

```diff
@@ -54,26 +54,26 @@ private FieldMasks() {
      * do not affect the execution result.
      *
      * @param mask     {@code FieldMask} to apply to each item of the input {@link Collection}.
-     * @param entities {@link Message}s to filter.
+     * @param messages {@link Message}s to filter.
      * @param typeUrl  Type of the {@link Message}s.
      * @return Non-null unmodifiable {@link Collection} of {@link Message}s of the same type that the input had.
```

Comment:
Please rewrite:

* Use `@Nonnull` instead if the first part of the statement.
* It is clear enough that this is a `Collection` of `Message` instances. And use also use generics to explicitly say the type is the same.

It's enough to describe that the collection returned contains elements with the field mask applied.